flask/app/analyses.py

```diff
@@ -124,6 +124,10 @@ def get_analysis(id: int):
                     "participant_type": dataset.tissue_sample.participant.participant_type,
                     "sex": dataset.tissue_sample.participant.sex,
                     "family_codename": dataset.tissue_sample.participant.family.family_codename,
+                    "updated_by": dataset.tissue_sample.updated_by_id
+                    and dataset.tissue_sample.updated_by.username,
```

Comment:
Is the purpose of `and` here and everywhere else to make sure the .username doesn't throw an error if `*_by` is undefined?

Reply:
sorry didn't see this yesterday, I believe so since the username can be returned without the id